app/core/model/src/main/java/io/syndesis/model/integration/IntegrationDeployment.java

```diff
@@ -36,388 +38,434 @@
+//        }
+//
+//        @JsonProperty("timesUsed")
+//        public Builder timesUsed(BigInteger timesUsed) {
+//            this.timesUsed = timesUsed;
+//            return this;
+//        }
+//
+//        @SuppressFBWarnings(value="EI_EXPOSE_REP")
+//        @JsonProperty("lastUpdated")
+//        public Builder lastUpdated(Date lastUpdated) {
+//            this.lastUpdated = lastUpdated;
+//            return this;
+//        }
+//
+//        @SuppressFBWarnings(value="EI_EXPOSE_REP")
+//        @JsonProperty("createdDate")
+//        public Builder createdDate(Date createdDate) {
+//            this.createdDate = createdDate;
+//            return this;
+//        }
+//
+//        public IntegrationDeployment build() {
+//            return new IntegrationDeployment(integrationId, version, spec, stepsDone, currentState, targetState, timesUsed, lastUpdated, createdDate);
+//        }
+//
+//        //Getters for Jackson
+//
+//
+//
+//        public Optional<Integer> getVersion() {
+//            return version;
+//        }
+//
+//
+//        public Optional<String> getIntegrationId() {
+//            return integrationId;
+//        }
+//
+//
+//        public String getName() {
+//            return name;
+//        }
+//
+//
+//
+//        public Integration getSpec() {
+//            return spec;
+//        }
+//
+//
+//        public List<String> getStepsDone() {
+//            return stepsDone;
+//        }
+//
+//
+//        public IntegrationDeploymentState getCurrentState() {
+//            return currentState;
+//        }
+//
+//
+//        public IntegrationDeploymentState getTargetState() {
+//            return targetState;
+//        }
+//
+//
+//        public BigInteger getTimesUsed() {
+//            return timesUsed;
+//        }
+//
+//        @SuppressFBWarnings(value="EI_EXPOSE_REP")
+//        public Date getLastUpdated() {
+//            return lastUpdated;
+//        }
+//
+//        @SuppressFBWarnings(value="EI_EXPOSE_REP")
+//        public Date getCreatedDate() {
+//            return createdDate;
+//        }
+//
+//        //For compatibility
+//        public Builder integrationId(Optional<String> integrationId) {
+//            this.integrationId = integrationId;
+//            return this;
+//        }
+//
+//        public Builder version(Optional<Integer> version) {
+//            this.version = version;
+//            return this;
+//        }
+//
+//        public Builder timesUsed(Optional<BigInteger> timesUsed) {
+//            this.timesUsed = timesUsed.orElse(BigInteger.ZERO);
+//            return this;
+//        }
+//
+//        public Builder createdDate(Optional<Date> createdDate) {
+//            this.createdDate = createdDate.orElse(new Date());
+//            return this;
+//        }
+//
+//        public Builder lastUpdated(Optional<Date> lastUpdated) {
+//            this.lastUpdated = lastUpdated.orElse(new Date());
+//            return this;
+//        }
+//
+//    }
```

Comment:
let's not do this, git versions source code and we have no need for commented out large blocks of code, remove it rather

Reply:
removing..